packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1495,12 +1495,14 @@ class EditableText extends StatefulWidget {
   /// Called for each tap that occurs outside of the[TextFieldTapRegion] group
   /// when the text field is focused.
   ///
-  /// If this is null, [FocusNode.unfocus] will be called on the [focusNode] for
-  /// this text field when a [PointerDownEvent] is received on another part of
-  /// the UI. However, it will not unfocus as a result of mobile application
-  /// touch events (which does not include mouse clicks), to conform with the
-  /// platform conventions. To change this behavior, a callback may be set here
-  /// that operates differently from the default.
+  /// If this is null, [EditableTextTapOutsideIntent] will be invoked. In the
+  /// default implementation, [FocusNode.unfocus] will be called on the
+  /// [focusNode] for this text field when a [PointerDownEvent] is received on
+  /// another part of the UI. However, it will not unfocus as a result of mobile
+  /// application touch events (which does not include mouse clicks), to conform
+  /// with the platform conventions. To change this behavior, a callback may be
+  /// set here or [EditableTextTapOutsideIntent] may be overwritten, that
+  /// operates differently from the default.
```

Comment:
Nit: "overwritten" => "overridden".

Also I think you can probably just get rid of "that operates differently from the default" and end the sentence there.